RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHCAL_cfi.py

```diff
@@ -65,3 +66,9 @@
 particleFlowClusterHCALOnly = particleFlowClusterHCAL.clone(
     clustersSource = "particleFlowClusterHBHEOnly"
 )
+
+#--- Use DB conditions for cuts&seeds for Run3 and Phase2
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+(phase2_common | run3_common).toModify( particleFlowClusterHCAL,
```

Comment:
How about creating a specific modifier for this change customization, and including that in the proper Era(s)?

Reply:
is there any example I can look at? 
I'd like to set `usePFThresholdsFromDB` to True only for HBHE (for run3 and phase2). 
And I did it using Modifier_phase2_common and Modifier_run3_common inside HBHE config files. 
Is it to be done differently?

Reply:
You could define a (e.g., you can find a better name instead) Modifier_run3_pfClusterHcal_cff.py in Configuration/Eras/python/, include it in Configuration/Eras/python/Era_Run3_cff.py, and use it to modify all configs here. Phase2 era starts from the Run3 one, and therefore once included in Era_Run3 there shouldn't be need to add it once again for Phase2

Reply:
Thanks Andrea. I've now implemented it. I chose a different name indeed, because this is not only for pfclusters (it is needed for both pfcluster and pfrechit); and since this will be used for run3 & phase2 both; with different threshold values, I did not mention run3 in the name. If the current name is not appropriate, I can change. Also, if people prefer to keep it inside Configuration/ProcessModifiers/python instead of Configuration/Eras/python, we can do that too